tests/unit/test_repos_repo.py

```diff
@@ -58,6 +58,64 @@ def test_asset_requires_a_positive_id(self):
 
         assert self.session.get.called is False
 
+    def test_create_fork(self):
+        """Verify the request to fork a repository."""
+        self.instance.create_fork()
+        self.post_called_with(
+            url_for('forks')
+        )
+
+    def test_create_hook(self):
+        """Verify the request to create a hook."""
+        data = {
+            'name': 'web',
+            'config': {
+                'url': 'http://example.com/webhook',
+                'content_type': 'json'
+            }
+        }
+
+        self.instance.create_hook(**data)
```

Comment:
Since we do some (lightweight) validation on the values passed to create_hook, can you add separate tests for the following cases:

- `name` is false-y (e.g., `None`, `''`), `config` is not false-y
- `name` is not false-y, `config` is false-y (e.g., `None`, `{}`, etc.)
- neither `name` nor `config` is false-y but `config` is not a dictionary

For each of those you want to do

```py
assert self.session.post.called is False
```